app/src/app/app.component.sass

```diff
@@ -16,6 +16,7 @@ footer
       img:not(#irit-logo)
         position: relative
         top: -4px
+        z-index: -1
```

Comment:
No. z-index must reflect how the boxes of the page are displayed in the axis perpendicular to the screen. Here we do not want some icons of the footer to stay behind the main part of the page (which always has z-index: 0). We want the information bar to be in front of the main part of the page.